src/pipelines.py

```diff
@@ -236,18 +257,15 @@ def mask_postprocessing(model, config, save_output=False):
                        adapter={'images': ([(model.name, 'multichannel_map_prediction')]),
                                 'target_sizes': ([('input', 'target_sizes')]),
                                 },
-                       cache_dirpath=config.env.cache_dirpath,
-                       save_output=save_output,
-                       cache_output=True)
+                       cache_dirpath=config.env.cache_dirpath, **kwargs)
```

Comment:
@jakubczakon Here we would like to cache mask_resize output and only this output, because we reuse it in score_builder. Other caches would only occupy memory. That's why I think we should set cache_output parameter separately, especially for mask_resize.

Reply:
@taraspiotr agreed, on it.